R/annotation.R

```diff
@@ -124,13 +160,17 @@ annotate_ssm_blacklist = function(mutations_df,
 }
 
 
-
```

Comment:
I would like you (or anyone else) to explain this function. Currently, it does not do anything, thus the return has been declared as "Nothing". Alternatively, I could remove the export for this function so that it doesn't populate the NAMESPACE.

Reply:
You can compare the dimensions of the data before and after this function is run. Note: to see what it is doing you need to get the maf that has not been deblacklisted before, otherwise your input will be identical to the output since this functionality is used in [lcr-modules](https://github.com/LCR-BCCRC/lcr-modules/blob/78c3ea60e0b8cc2c27595b6dc7041aff2b71d9f9/modules/vcf2maf/1.3/vcf2maf.smk#L221-L244) to deblacklist mafs at the first place.